pkgs/shells/ion/default.nix

```diff
@@ -4,26 +4,23 @@ with rustPlatform;
 
 buildRustPackage rec {
   pname = "ion";
-  version = "1.0.5";
+  version = "unstable-2020-03-22";
 
   src = fetchFromGitHub {
     owner = "redox-os";
     repo = "ion";
-    rev = version;
-    sha256 = "0i0acl5nw254mw8dbfmb4792rr71is98a5wg32yylfnlrk7zlf8z";
+    rev = "1fbd29a6d539faa6eb0f3186a361e208d0a0bc05";
+    sha256 = "0r5c87cs8jlc9kpb6bi2aypldw1lngf6gzjirf13gi7iy4q08ik7";
   };
 
-  cargoSha256 = "0f266kygvw2id771g49s25qsbqb6a0gr1r0czkcj96n5r0wg8wrn";
+  cargoSha256 = "1ph3r3vspy700mb8pica8478v9arqz07k2nzpbrdkdkqgfcwlgcg";
 
   meta = with stdenv.lib; {
     description = "Modern system shell with simple (and powerful) syntax";
-    homepage = https://github.com/redox-os/ion;
+    homepage = "https://gitlab.redox-os.org/redox-os/ion";
     license = licenses.mit;
     maintainers = with maintainers; [ dywedir ];
     platforms = platforms.all;
-    # This has not had a release since 2017, and no longer compiles with the
-    # latest Rust compiler.
-    broken = false;
```

Comment:
This was supposed to say `broken = true;`, but at any rate now it's actually not broken :)